src/providers/push/apns/apns.provider.ts

```diff
@@ -48,8 +48,9 @@ export class ApnsProvider implements Provider<any> {
   }
   getMainNote(message: ApnsMessage) {
     const note = new apns.Notification();
-    note.expiry = Math.floor(Date.now() / 1000) + 3600; // Expires 1 hour from now.
-    note.badge = this.apnsConfig?.options.badge ?? 3;
+    note.expiry = Math.floor(Date.now() / 1000) + 3600; //NOSONAR
```

Comment:
instead of //nosonar define constants here and replace those

Reply:
done changes